packages/admin/admin-date-time/src/TimePicker.styles.ts

```diff
@@ -0,0 +1,18 @@
+import { InputWithPopperClassKey } from "@comet/admin";
+import { createStyles } from "@mui/styles";
+
+import { TimePickerProps } from "./TimePicker";
+
+export type TimePickerClassKey = InputWithPopperClassKey;
+
+export const styles = () =>
+    createStyles<TimePickerClassKey, TimePickerProps>({
+        root: {},
+        inputBase: {},
+        popper: {},
+        paper: {
+            minWidth: 110,
+            height: 324,
```

Comment:
The `height` seems rather arbitrary. What's your reasoning behind it?

Reply:
This is wrong 😅
It should show exactly 7 menu-items, I fixed this in [740047a](https://github.com/vivid-planet/comet-admin/pull/639/commits/740047a30724d857b2c38bceb2341986b8d511a8).